### DISCUSSION TITLE
How do I fix Hitbox?

### DISCUSSION THREAD

**Discussion by SamGamerYT** [Kas-tle/java2bedrock.sh#1709] [Converter Help]
How do I fix hitbox? A bunch of the converted items hitboxes are messed up now

**Comment by Kas-tle** [comment]
Items don't have hitboxes.

**Comment by SamGamerYT** [comment]
I meant the models that are converted.

**Reply by Kas-tle** [reply]
Hitboxes have nothing to do with the resource pack. The server sets the hitbox of an entity, not the client.

**Comment by SamGamerYT** [comment]
Another thing I have notice is that for bedrock players the pack loading time is horrendous

**Reply by Kas-tle** [reply]
Then include fewer models and decrease texture resolution.
